packages/ui/src/models/camel/camel-route.ts

```diff
@@ -0,0 +1,62 @@
+import { CamelResource } from './camel-resource';
+import { BaseCamelEntity } from './entities';
+import { CamelRouteVisualEntity, isCamelRoute } from '../visualization/flows';
+import { BeansEntity, isBeans } from '../visualization/metadata';
+import { SourceSchemaType } from './source-schema-type';
+import { isDefined } from '../../utils';
+
+export class CamelRoute implements CamelResource {
```

Comment:
Would it help to provide a suffix to this entity's kind, so it's easier to distinguish between this and the CamelRouteVisualEntity?

Maybe something like `CamelRouteResouce`, `KameletBindingResource`, `PipeResource`, and likewise for the file name.

Reply:
done :+1: